docs/architecture/accounts.md

```diff
@@ -18,4 +18,4 @@ See [Smart contracts](../smart-contracts).
 - Smart Rollup accounts are another type of originated account.
```

Comment:
As you said just above that smart contracts = originated accounts, you cannot say here that smart rollups are another type of originated accounts. Maybe rather "programmable accounts", or "accounts containing code and storage"?

Reply:
I don't think that smart rollups are a kind of originated account. In fact, they are not at all accounts. So I think:
- you should revert the change to smart contracts (which are the same thing as "originated account")
- either rename this list as "addresses" instead of "accounts", or remove smart rollups from the list

Reply:
I'm removing changes to this file from this PR and I'm working on clarifying accounts and addresses in this pr: https://github.com/trilitech/tezos-developer-docs/pull/252